package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "hangoutsidesf",
+  "version": "1.0.0",
+  "description": "[Contributing Guidelines](CONTRIBUTING.md)",
+  "main": "server/index.js",
+  "scripts": {
+    "build": "webpack",
```

Comment:
needs to use a prod webpack.config since current one has hotmiddleware as an entry

Reply:
Created a production webpack config file without hod middleware.